src/components/common/Extensions/UserHub/partials/UserHubContent/UserHubContent.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { FC, useState } from 'react';
+import clsx from 'clsx';
+
+import { useMobile } from '~hooks';
+import ReputationTab from '../ReputationTab';
+import StakesTab from '../StakesTab';
+import TransactionsTab from '../TransactionsTab';
+import Icon from '~shared/Icon';
+import { tabList } from './consts';
+import { UserHubNavigationItem } from './types';
+import TitleLabel from '~v5/shared/TitleLabel';
+import { stakesMock } from '../StakesTab/consts';
+import Select from '~v5/common/Fields/Select';
+import { formatText } from '~utils/intl';
+import ButtonLink from '~v5/shared/Button/ButtonLink';
```

Comment:
The imports here are all mixed can we separate them so that they follow the format in the rest of the app eg.

import Select from '~v5/common/Fields/Select';

import { stakesMock } from '../StakesTab/consts';

Reply:
@thomassdk done